src/pybind/mgr/dashboard/HACKING.rst

```diff
@@ -1090,16 +1090,18 @@ How to access the manager module instance from a controller?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 We provide the manager module instance as a global variable that can be
-imported in any module. We also provide a logger instance in the same way.
+imported in any module.
 
 Example:
 
 .. code-block:: python
 
+  import logging
   import cherrypy
-  from .. import logger, mgr
+  from .. import mgr
   from ..tools import ApiController, RESTController
 
+  logger = logging.getLogger('controllers.servers')
```

Comment:
What about just using `__name__` as a good default here? 
```suggestion
  logger = logging.getLogger(__name__)
```

This would prevent copy&paste errors in the future. And would make this logging a child-logger of dashboard and thus inheriting the parent's properties.

Reply:
> I initially used `__name__` but `__name__` also included the `dashboard.` prefix and makes the string too long for my preference when reading the log file.

Probably the convention to use `__name__` is because the logger will be in a hierarchy, so descendants loggers can inherit handlers from ancestors if the propagate attribute is true. 

For example, suppose we want to fine-tune the logging behavior of `dashboard.services` and its descendants, we can config `dashboard.services` logger instance. Then its descendants will share the same logging behavior by default. If the logger instance in `dashboard.services.auth` module is initiated with name `auth` (which is actually create a `root.auth` logger), it can't use the logger from `dashboard.services` module we just defined.

Possible ways to shorten the name in log is to use a custom formatter or to use `%(module)s` attribute in default formatter (although it only displays the single module name).

Reply:
Actually, I'll not tweek the formatter. I'll just leave the formatter as it is, and will use `__name__` throughout the dashboard code.